browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -828,58 +793,17 @@ void VerticalTabStripRegionView::Layout() {
       {contents_bounds.x(),
        contents_bounds.bottom() - new_tab_button_->height()});
 
-  if (base::FeatureList::IsEnabled(
-          tabs::features::kBraveVerticalTabsStickyPinnedTabs)) {
-    const gfx::Size header_size{contents_bounds.width(),
-                                tabs::kVerticalTabHeight + kHeaderInset * 2};
-    header_view_->SetPosition(contents_bounds.origin());
-    header_view_->SetSize(header_size);
-
-    contents_view_->SetSize(
-        {contents_bounds.width(), contents_bounds.height() -
-                                      new_tab_button_->height() -
-                                      header_view_->height()});
-    contents_view_->SetPosition({contents_bounds.origin().x(),
-                                 header_view_->y() + header_view_->height()});
-    UpdateOriginalTabSearchButtonVisibility();
-
-    // Put resize area on the right side, overlapped with contents.
-    constexpr int kResizeAreaWidth = 4;
-    resize_area_->SetBounds(width() - kResizeAreaWidth, contents_bounds.y(),
-                            kResizeAreaWidth, contents_bounds.height());
-    return;
-  }
-
-  // 2. ScrollView takes the rest of space.
-  // Set preferred size for scroll view to know this.
   const gfx::Size header_size{contents_bounds.width(),
                               tabs::kVerticalTabHeight + kHeaderInset * 2};
-  header_view_->SetPreferredSize(header_size);
+  header_view_->SetPosition(contents_bounds.origin());
   header_view_->SetSize(header_size);
 
-  scroll_view_->SetSize({contents_bounds.width(),
-                         contents_bounds.height() - new_tab_button_->height()});
-  scroll_view_->SetPosition(contents_bounds.origin());
-
-  auto scroll_viewport_height = scroll_view_->height() - header_size.height();
-  if (scroll_view_->GetMaxHeight() != scroll_viewport_height) {
-    scroll_view_->ClipHeightTo(0, scroll_viewport_height);
-  }
-
-  if (base::FeatureList::IsEnabled(features::kScrollableTabStrip) &&
-      tabs::utils::ShouldShowVerticalTabs(browser_)) {
-    contents_view_->SetSize({scroll_view_->width(), scroll_view_->height()});
-    auto* nested_scroll_view = GetTabStripScrollContainer()->scroll_view_.get();
-    nested_scroll_view->SetSize(
-        {scroll_view_->width(), scroll_viewport_height});
-    nested_scroll_view->ClipHeightTo(0, scroll_viewport_height);
-  } else {
-    contents_view_->SetSize(
-        {scroll_view_->width(),
-         std::max(scroll_viewport_height,
-                  contents_view_->GetPreferredSize().height())});
-  }
-
```

Comment:
a lot of things is going here. I assume everything is intended and works as expected here.

Reply:
Yeah, I checked this works well. This changes removing unreachable code below line 853, when sticky pinned tab feature is eanbled.